recipes/openmm/conda_build_config.yaml

```diff
@@ -0,0 +1,15 @@
+channel_targets:
+- conda-forge testing
+cuda_compiler:
+- nvcc
+cuda_compiler_version:
+# - 9.2
+- 10.0
+# - 10.1
+docker_image:
+# - condaforge/linux-anvil-cuda:9.2
+- condaforge/linux-anvil-cuda:10.0
+# - condaforge/linux-anvil-cuda:10.1
```

Comment:
You can uncomment this in the feedstock. If you want to add more CUDA packages through staged-recipes, we'd appreciate some help in getting staged-recipes to play nicely with CUDA packages.

Reply:
Have been meaning to raise an issue about this. Submitted issue ( https://github.com/conda-forge/staged-recipes/issues/9773 ) for discussion and tracking work on this task.

Reply:
Following issue!